docs/upgrade.html

```diff
@@ -66,7 +66,10 @@
     <li> ProduceRequest/Response v2 has been introduced and it is used by default to support message format 0.10.0 </li>
     <li> FetchRequest/Response v2 has been introduced and it is used by default to support message format 0.10.0 </li>
     <li> MessageFormatter interface was changed from <code>void writeTo(byte[] key, byte[] value, PrintStream output)</code> to
-        <code>void writeTo(byte[] key, byte[] value, long timestamp, TimestampType timestampType, PrintStream output)</code> </li>
+        <code>def writeTo(consumerRecord: ConsumerRecord[Array[Byte], Array[Byte]], output: PrintStream)</code> </li>
```

Comment:
Discussed this with Jun. Even though it many implementors may actually use Java, we decided that it was better to be consistent with the actual code to avoid confusion (people might look for these interfaces in our codebase and would not find them). And since people need to understand Scala anyway to implement the interfaces, it's not too much of an issue.